course_catalog/etl/loaders_test.py

```diff
@@ -562,8 +563,16 @@ def test_load_playlist(mock_tasks):
 def test_load_playlists_unpublish():
     """Test load_playlists when a video/playlist gets unpublished"""
     channel = VideoChannelFactory.create()
-    playlist1 = PlaylistFactory.create(channel=channel, published=True)
-    playlist2 = PlaylistFactory.create(channel=channel, published=True)
+
+    userlist1 = UserListFactory.create()
+    userlist2 = UserListFactory.create()
+
+    playlist1 = PlaylistFactory.create(
+        channel=channel, published=True, has_user_list=True, user_list=userlist1
+    )
+    playlist2 = PlaylistFactory.create(
+        channel=channel, published=True, has_user_list=True, user_list=userlist2
+    )
```

Comment:
Not really necessary, but what do you think of adding another playlist to [check one more condition](https://github.com/mitodl/open-discussions/pull/2749/files#diff-fd3cb62a519d4bd6c7e4fe584456e70bR456):
```
    playlist3 = PlaylistFactory.create(
        channel=channel, published=True, has_user_list=True, user_list=None
    )
```
and then adding `assert` statements for that playlist below?

Reply:
Done!